.github/SECURITY.md

```diff
@@ -2,7 +2,10 @@ GitHub takes the security of our software products and services seriously, inclu
 
 If you believe you have found a security vulnerability in GitHub CLI, you can report it to us in one of two ways:
 
-* Report it to this repository directly using [private vulnerability reporting][]. Such reports are not eligible for a bounty reward.
+* Report it to this repository directly using [private vulnerability reporting][].
```

Comment:
What is this syntax?

```
[private vulnerability reporting][]
```

It seems to work but I was surprised.

Reply:
I had the same question in another PR. He answered [here](https://github.com/cli/cli/pull/9683#discussion_r1783635991).

Reply:
Thank you @jtmcg 🙏 

I wish I could find some github docs on it, but there doesn't seem to be any 🤷  🤔

Reply:
> I wish I could find some github docs on it, but there doesn't seem to be any 🤷 🤔

This feature is documented (obscurely) [here]. There's a whole breadth of interesting syntax patterns that can be used in GFM, including the text that shows when you hover over a link (including in this very comment 😉).

[here]: https://github.github.com/gfm/#link-reference-definitions "Ohh look hover text!"

Reply:
Thanks!